### DISCUSSION TITLE
Edge Function GET call from 1 supabase project to another causing a 500 error

### DISCUSSION THREAD

**Discussion by pruthvi2103** [supabase/supabase#23146] [Questions] [answered]
I have 2 projects A and B and have 2 edge functions Afunc and Bfunc. My business logic is that i call Bfunc and it calls Afunc via fetch behind the scenes and does some operations on it and stores it to the DB. Thing is today i notice i am getting a 500 error for the Afunc call. Afunc calls works without issues with the same headers via curl and hoppscotch only thing i see is different is `      "user_agent": "supabase-edge-runtime"
`

is this disabled from supabase side?

**Comment by GaryAustin1** [comment]
Do you know what the error is?   getUser() has been getting errors with supabase-js 2.42.5 in edge functions.

**Reply by GaryAustin1** [reply]
https://status.supabase.com/incidents/k5g20mlgcp9r

**Reply by etienne678** [reply]
have the same problem, also get 500 errors on eu-west.  it is not self healed for me.

**Reply by Voqera** [reply]
getting 500 on my us-west edge functions too. Ive also deployed a new version of the script that was returning 500 errors and the logs are not being published through the dashboard for that edge function (as of 1016a PT)

**Reply by GaryAustin1** [reply]
Make sure you have support requests in.

**Reply by Voqera** [reply]
yep, support request is in but also wanted to see if it was me doing/not doing something because of a change I wasn't aware of

**Reply by GaryAustin1** [reply]
As I said check if you are getting an error from getUser().  A supabase-js change was made that is breaking it in edge functions depending if you called it without getUser(jwt).

**Reply by Voqera** [reply]
I can't tell if I'm getting an error because my functions are generating any logs

**Reply by GaryAustin1** [reply]
Yeah seems like the outage then.

**Reply by pruthvi2103** [reply]
It was indeed an outage. It works without issues today

**Comment by etienne678** [comment]
for me, it is not the getUser() issue. i had that issue as well yesterday, then downgraded to supabase-js@2.42.4 and i can get the user. but today i still get the 500 error. It is also not occuring on getUser() but somewhere else, so i think those two issues are not directly connected.

**Reply by GaryAustin1** [reply]
Yeah, just confusing here and discord as users are hitting both things.

**Comment by pruthvi2103** [comment, marked as answer]
Got an answer from support team, seems it was an outage from their side.Normally ran the function today and it works!
